stdlib/license.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3.6
+# -*- coding: utf-8 -*-
+"""An enumeration of possible a softwares' licenses."""
```

Comment:
Should be `of possible softwares' licenses`.

Reply:
`Software` is uncountable, so more like `of possible software licenses`